builder/lxd/config.go

```diff
@@ -19,6 +19,9 @@ type Config struct {
 	// name.
 	OutputImage   string `mapstructure:"output_image" required:"false"`
 	ContainerName string `mapstructure:"container_name"`
+	// The (optional) name of the LXD remote on which to publish the
+	// container image.
+	PublishRemote string `mapstructure:"publish_remote" required:"false"`
```

Comment:
Isn't `remote_name` better? `PublishRemote` sounds more like a boolean flag to me.
```suggestion
	RemoteName string `mapstructure:"remote_name" required:"false"`
```

Reply:
Yes, `publish_remote` does indeed sound like it should be a boolean.
However, `remote_name` could be misunderstood as also applying to the container launch process (which it doesn't affect). How about `publish_remote_name`?